packages/common/src/index.ts

```diff
@@ -1194,6 +1204,18 @@ export const getItemColor = (item: Field | TableCalculation) => {
     return '#0A6640';
 };
 
+export const getResultValues = (
+    rows: ResultRow[],
+    onlyRaw: boolean = false,
+): { [col: string]: any }[] =>
+    rows.map((row: ResultRow) => Object.keys(row).reduce((acc, key) => {
+            const value: string = onlyRaw
+                ? row[key]?.value?.raw
+                : row[key]?.value?.formatted || row[key]?.value?.raw;
```

Comment:
Why would formatted be empty but not raw?

Reply:
Am I right in saying that `value` can not be undefined? just looking at the type of `ResultRow` above